Tasks/UsePythonVersionV0/usepythonversion.ts

```diff
@@ -95,11 +91,23 @@ async function useCpythonVersion(parameters: Readonly<TaskParameters>, platform:
         task.warning(task.loc('PythonVersionRetirement'));
     }
 
-    if (isExactVersion(semanticVersionSpec)) {
-        task.warning(task.loc('ExactVersionNotRecommended'));
+    let installDir: string | null = tool.findLocalTool('Python', semanticVersionSpec, parameters.architecture);
+    // Python version not found in local cache, try to download and install
+    if (!installDir) {
+        task.debug(`Could not find a local python installation matching ${semanticVersionSpec}.`);
+        try {
+            task.debug('Trying to download python from registry.');
+            await installPythonVersion(semanticVersionSpec, parameters);
```

Comment:
Just as an idea - could it make sense to add input to be able to turn off download from github registry? Are there some cases when users would prefer to use local python only?

Reply:
Added a new input for this.
This could be useful if the customer does not want any downloads to be triggered.